test/test_utils/__init__.py

```diff
@@ -130,6 +133,7 @@ def get_ami_id_ssm(region_name, parameter_path):
     UBUNTU_18_BASE_DLAMI_US_EAST_1,
```

Comment:
remove the reference from here and declaration if it is not used in conftest

Reply:
done